pkg/model/ch_config_generator.go

```diff
@@ -110,10 +110,14 @@ func (c *ClickHouseConfigGenerator) GetHostZookeeper(host *chiv1.ChiHost) string
 		// <node>
 		//		<host>HOST</host>
 		//		<port>PORT</port>
+		//		<secure>%d</secure>
 		// </node>
 		util.Iline(b, 8, "<node>")
 		util.Iline(b, 8, "    <host>%s</host>", node.Host)
 		util.Iline(b, 8, "    <port>%d</port>", node.Port)
+		if node.Secure == 1 {
```

Comment:
making bool out of int looks a little awkward

Reply:
Changed to IsSecure() method